File: web/src/app/models/statistics.ts
export class Statistics {
  constructor(
    public name?: string,
    public numberOfRace?: number,
    public numberOfWin?: number,
    public numberOfPodium?: number,
    public numberOfChampion?: number,
    public sumPoint?: number,
    public seasonStatistics?: SeasonStatistics[],
    public positionStatistics?: PositionStatistics[]
  ) {}
}

export class SeasonStatistics {
  constructor(
    public name?: string,
    public teamName?: string,
    public teamColor?: string,
    public position?: number
  ) {}
}

export class PositionStatistics {
  constructor(
    public position?: string,
    public number: number = 0
  ) {}
}

File: api/Models/Statistics.cs
using System.Text.Json.Serialization;

namespace api.Models
{
    public class Statistics {
        public string? Name { get; set; }
        public int? NumberOfRace { get; set; }
        public int? NumberOfWin { get; set; }
        public int? NumberOfPodium { get; set; }
        public int? NumberOfChampion { get; set; }
        public double? SumPoint { get; set; }
        public List<SeasonStatistics>? SeasonStatistics { get; set; }
        public List<PositionStatistics>? PositionStatistics { get; set; }
    }

    public class SeasonStatistics {
        public string? Name { get; set; }
        public string? TeamName { get; set; }
        public string? TeamColor { get; set; }
        public DateTime CreatedAt { get; set; }
        public int? Position { get; set; }
    }

    public class PositionStatistics {
        public string? Position { get; set; }
        public int? Number { get; set; }
    }
}